### DISCUSSION TITLE
Idea for "balanced" pvp at lvl 70

### DISCUSSION THREAD

**Discussion by Murdarahh** [chromiecraft/chromiecraft#4702] [Suggestions]
Guys i think i have a nice idea how to "balance"lvl 70 pvp.
Since we have wotlk talents i dont think lvl 70 resiliance cap will be enough, so here is my plan:
GMs shoud give us a custum vendor where we can buy a wotlk rare quality gem "Mystic Autumn's Glow" wich give us + 16 resiliance. But only that gem and nothing else. If u like it u can think about it.

**Comment by noscripted** [comment]
It would be enough if the server got rid of unfair pvp players who have "improvements". Every single pvp player should be verified because there are too many cheaters in such a small pvp community. Let's not pretend that all pvp players are honest...

**Reply by heyitsbench** [reply]
What cheats are being used on CC in PvP? First I'm...ever hearing of it.

**Comment by balleny** [comment]
please, try to keep the discussions together
https://github.com/chromiecraft/chromiecraft/discussions/4161
